test/unit/callout.handlers/index.test.ts

```
@@ -0,0 +1,163 @@
+    const callout = {
+      settings: {
+        contribution: {
+          allowedTypes: [
+            CalloutContributionType.Link,
+            CalloutContributionType.Post,
+          ],
+        },
+      },
+    };
+
+    const result = await handleCallout(callout as any, logger, null);
```

Biome (2.4.9)

[error] This variable result is unused.

(lint/correctness/noUnusedVariables)